es/_posts/2025-09-05-058.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+pretitle: Fazendo um 
+title: commit
+command: git commit
+descriptors:
+  - command: Comando para criar um commit
+info: Esse comando abrirá um editor de texto\npara escrita da mensagem de commit
```

Comment:
@R055A I believe you forgot to either commit and/or push the Spanish version hehe can you do that please?